presentation/common/src/presentation-common/CommonLoggerCategory.ts

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Core
```

Comment:
@grigasp @saskliutas this was marked "@module Logging" which docs build did not like because no such module is defined in presentation-common.ts.
I didn't see any reason to introduce a new Logging module to hold a single enum with a single value so I changed it to "Core".